cli/azd/pkg/password/shuffle.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package password
+
+import "math/rand"
```

Comment:
Would prefer it if we use `crypto/rand` here.  If we are worried about the method becoming error returning, I would just say that we should panic if we can't read from `crypto/rand`.

Reply:
Changed. It is a bit of an overkill for shuffling but hey, it is good to be on the safe side.